tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,238 @@
+# -*- coding: utf-8 -*-
 """
-.. _tut-annotations:
-
-The :term:`Events <events>` and :class:`~mne.Annotations` data structures
-=========================================================================
-
-:term:`Events <events>` and :term:`annotations` are quite similar.
-This tutorial highlights their differences and similarities, and tries to shed
-some light on which one is preferred to use in different situations when using
-MNE.
-
-Both events and :class:`~mne.Annotations` can be seen as triplets
-where the first element answers to **when** something happens and the last
-element refers to **what** it is.
-The main difference is that events represent the onset in samples taking into
-account the first sample value
-(:attr:`raw.first_samp <mne.io.Raw.first_samp>`), and the description is
-an integer value.
-In contrast, :class:`~mne.Annotations` represents the
-``onset`` in seconds (relative to the reference ``orig_time``),
-and the ``description`` is an arbitrary string.
-There is no correspondence between the second element of events and
-:class:`~mne.Annotations`.
-For events, the second element corresponds to the previous value on the
-stimulus channel from which events are extracted. In practice, the second
-element is therefore in most cases zero.
-The second element of :class:`~mne.Annotations` is a float
-indicating its duration in seconds.
-
-See :ref:`ex-read-events`
-for a complete example of how to read, select, and visualize **events**;
-and :ref:`tut-artifact-rejection` to
-learn how :class:`~mne.Annotations` are used to mark bad segments
-of data.
-
-An example of events and annotations
-------------------------------------
-
-The following example shows the recorded events in `sample_audvis_raw.fif` and
-marks bad segments due to eye blinks.
+Parsing events from raw data
+============================
+
+This tutorial describes how to read experimental events from raw recordings,
+and how to convert between the two different representations of events within
+MNE-Python (Events arrays and Annotations objects).
+
+.. contents:: Page contents
+   :local:
+   :depth: 1
+
+In the :ref:`introductory tutorial <overview-tut-events-section>` we saw an
+example of reading experimental events from a :term:`"STIM" channel <stim
+channel>`; here we'll discuss :term:`events` and :term:`annotations` more
+broadly, give more detailed information about reading from STIM channels, and
+give an example of reading events that are included in the data file as an
+embedded array.
```

Comment:
I've been trying to keep the text before `page contents` very short, because it appears when you hover over the tutorial thumbnail in the gallery.  So I prefer not to merge the paragraphs.  But you make a good point about forward-referencing the other tutorials. I added this:

```
The tutorials :doc:`../raw/plot_events` and
:doc:`../raw/plot_annotating_raw` discuss how to plot, combine, load, save, and
export :term:`events` and :class:`~mne.Annotations` (respectively), and the
latter tutorial also covers interactive annotation of :class:`~mne.io.Raw`
objects.
```

Reply:
Maybe mine was bit too long.

If `page contents` appears when hovering then we should make sure that the description describes the tutorial and keep the links further to not appear when hover.

Reply:
@massich the "hover text" automatically stops at a paragraph boundary, so there is no risk of the page contents ending up in there. I think what I have now in these tutorials is good (still gets truncated, but gives the main gist of the tutorial).

BTW, this is a good place to mention that API cross-refs should not occur in that first sentence or two, because they appear in raw form in the hover text, and that API cross-refs should also be left out of section titles (because they mess up the TOC links).

Reply:
> API cross-refs should not occur in that first sentence or two, because they appear in raw form in the hover text

Probably worth a SG issue to use regex or something to remove Sphinx markup from the hover text

Reply:
done: sphinx-gallery/sphinx-gallery#497